Social Distancing Measures.ipynb

```diff
@@ -0,0 +1,386 @@
+{
```

Comment:
<p>What is the thought process for <code>initE</code> to be zero?</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NirantK/coronaIndia/pull/26/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>I have added a new commit wrt this. Have a look. </p>